DSharpPlus/DiscordClient.cs

```diff
@@ -2634,8 +2634,7 @@ public override void Dispose()
             this._guilds = null;
             this._heartbeatTask = null;
             this._privateChannels = null;
-            this._webSocketClient.DisconnectAsync().ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Not necessary to remove the line, simply wrap both of them into a null check.

Reply:
I removed this because the disconnect call is completely redundant (it is done in both the disconnect of the DiscordClient and the dispose of the WebSocketClient) and there is no check that stops the disconnect if it has been executed previously.
The use of the ? was done because the same was done for the cancelTokenSource ~10 lines above this change.
If it is really preferred I'll add the line back and wrap it in an if statement😃.